resources/org.ebnf

```diff
@@ -151,7 +151,7 @@ keyword-value = anything-but-newline
 
 (* TODO allow empty properties with or without trailing space *)
 (* TODO looks like node-property-line also parses :END: *)
-node-property-line = ! <':END:> <':'> node-property-name [node-property-plus] <':'> ( <' '> node-property-value | [<' '>] ) <eol>
+node-property-line = ! <':END:'> <':'> node-property-name [node-property-plus] <':'> ( <' '> node-property-value | [<' '>] ) <eol>
```

Comment:
Nice, this was a typo. Thanks!

Reply:
You're welcome. Good work on the PR so far!

I've asked my business partner on feedback on how to work with semantic elements and he's writing something up as I type this^^